website/src/pages/docs/guides/graphql-server-apollo-yoga-with-server-preset.mdx

```diff
@@ -374,11 +408,11 @@ export type ResolversParentTypes = {
 ```ts filename="src/schema/user/resolvers/User.ts" {3-5,7-10}
 import type { UserResolvers } from './../../types.generated'
 export const User: UserResolvers = {
-  fullName: () => {
+  fullName: async (_parent, _arg, _ctx) => {
     /* User.fullName resolver is required because User.fullName exists but UserMapper.fullName does not */
   },
 
-  isAdmin: ({ isAdmin }) => {
+  isAdmin: ({ isAdmin }, _arg, _ctx) => {
```

Comment:
do we need to this change? 🤔

Reply:
The server preset now generates this by default, so I think matching it here helps set the user expectation